### ISSUE TITLE
Line formatter deletes quotes at the ends of lines

### ISSUE THREAD

**Issue by jakebailey** [Microsoft/python-language-server#776]
There may not be a token showing up for these unfinished quotes, which causes them to be deleted.

![triple](https://user-images.githubusercontent.com/5341706/54548865-10de3280-4966-11e9-9242-7efff8505ddc.gif)

![single](https://user-images.githubusercontent.com/5341706/54548857-0e7bd880-4966-11e9-8b4a-0618e5c6863d.gif)

**Comment by fireattack** [comment]
Yeah, this is especially annoying when using triple-quoted string literals.

Keep in mind even if you typed something after the quote, it will still be deleted after enter/new line. E.g.:

1. type `a= '''my first line`
2. press enter

the `'''` will be deleted.
